lms/lib/comment_client/utils.py

```diff
@@ -171,3 +172,19 @@ def __init__(self, collection, page, num_pages, thread_count=0, corrected_text=N
         self.num_pages = num_pages
         self.thread_count = thread_count
         self.corrected_text = corrected_text
+
+
+def check_forum_heartbeat():
```

Comment:
If the forums are disabled via `ForumsConfig` we probably want to skip this check and simply return success.  Of course it wouldn't make much sense to configure this check and then disable the forums, but it might be useful for maintenance or similar.

Reply:
Ok, added.